events/bingo/bingoCardApp/ui/css/bingoCard_ui.css

```diff
@@ -1,39 +1,11 @@
 body {
-    margin: 10;
-    width: 100% - 10px;
+    margin: 0;
+    width: 100%;
     font-family: 'Raleway', sans-serif;
     color: #cccbcb;
     background: linear-gradient(#2b2b2b, #303030);
 }
 
-.top-bar {
-    height: 90px;
-    background: linear-gradient(#2b2b2b, #303030);
-    font-weight: bold;
-    padding-left: 30px;
-    padding-right: 30px;
-    display: flex;
-    align-items: center;
-    position: fixed;
-    width: 480px;
-    top: 0;
-    z-index: 1;
-}
-
-.content {
-    margin-top: 90px;
-    padding: 30px;
-}
-
-td {
-    width: 20%
-}
-
-table {
-    width: 100%;
-    text-align: center
-}
-
 input[type=button] {
     font-family: 'Raleway';
     font-weight: bold;
```

Comment:
blank line after this one(formatting consistency)

Reply:
That is if it's not intentional, it probably is so ignore

Reply:
intentional indeed